arkworks/src/kzg_types.rs

```diff
@@ -36,6 +46,16 @@ use std::ops::{AddAssign, Mul, Neg, Sub};
 
 extern crate alloc;
 use alloc::sync::Arc;
+use std::ptr;
```

Comment:
Try to avoid importing from `std`, to be able compiling for `no_std` environments.

You can replace this with importing from `core`:
```suggestion
use core::ptr;
```